src/main/scala/com/databricks/spark/csv/CsvRelation.scala

```diff
@@ -124,13 +124,14 @@ case class CsvRelation protected[spark] (
           case aiob: ArrayIndexOutOfBoundsException if permissive =>
             (index until schemaFields.length).foreach(ind => rowArray(ind) = null)
             Some(Row.fromSeq(rowArray))
-          case nfe: java.lang.NumberFormatException if dropMalformed =>
+          case _: java.lang.NumberFormatException |
+               _: IllegalArgumentException if dropMalformed =>
```

Comment:
Is this newline appropriate? I cannot find a proper document in [databricks/scala-style-guide](https://github.com/databricks/scala-style-guide).